src/routes/Send/Send.tsx

```diff
@@ -48,12 +49,12 @@ const Information: FC = memo(() => {
   )
 })
 
-const hasEnoughIron = (balance: bigint, amount: bigint, fee: bigint) => {
+const hasEnoughIron = (balance: bigint, amount: bigint, fee = BigInt(0)) => {
   const zero = BigInt(0)
   if (balance === zero) {
     return false
   }
-  return (amount || amount === zero) && (fee || fee === zero)
+  return balance && (amount || amount === zero) && (fee || fee === zero)
```

Comment:
question: Are the types correct here? The way I understand BigInt, only 0 is falsy so these conditions are the same on the left and right since it's like saying `(thing !== zero || thing === zero)`.